rasa/cdu/command_processor.py

```diff
@@ -358,8 +373,31 @@ def clean_up_commands(
             structlogger.debug(
                 "command_executor.skip_command.slot_already_set", command=command
             )
+        elif isinstance(command, SetSlotCommand) and command.name not in slots_so_far:
+            all_questions = [
+                s for f in all_flows.underlying_flows for s in f.get_question_steps()
+            ]
+            use_slot_fill = True
+            for q in all_questions:
+                if q.question == command.name and not q.skip_if_filled:
+                    use_slot_fill = False
```

Comment:
this could potentially break if some generic slot is used with `skip_if_filled: True` in one flow and with `skip_if_filled: False` in another, but not sure anything can be done about it.

Reply:
mhm agreed but don't think we can resolve that can we